media/css/newsletter/newsletter-firefox.less

```diff
@@ -0,0 +1,248 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import (reference) '../pebbles/lib.less';
+@import '../pebbles/components/footer.less';
+@import '../pebbles/elements/forms.less';
+
+@color-fox-purple: #3a304b;
+@color-link-blue: #00a7e0;
+
+.content {
+    .clearfix;
+
+    @media @mq-tablet {
+        width: @width-tablet;
+    }
+
+    @media @mq-desktop {
+        width: @width-desktop;
+    }
+}
+
+.section-intro {
+    background-color: @color-fox-purple;
+
+    .content {
+        background-image: url('/media/img/newsletter/firefox/mailbox.svg'),
+                          url('/media/img/newsletter/firefox/fox-tail.svg');
+        background-position: bottom -40px left 10px, bottom -45px right -10px;
+        background-repeat: no-repeat, no-repeat;
+        -webkit-background-size: 141px 157px, 153px 115px;
+                background-size: 141px 157px, 153px 115px;
+        height: 167px;
+        width: @width-phone;
+
+        @media @mq-phone-wide {
+            background-position: bottom -60px left, bottom -65px right;
+            -webkit-background-size: 215px 239px, 230px 173px;
+                    background-size: 215px 239px, 230px 173px;
+            height: 229px;
+            width: @width-phone-wide;
+        }
+
+        @media @mq-tablet {
+            background-position: bottom -90px left 40px, bottom -90px right 40px;
+            -webkit-background-size: 300px 333px, 320px 241px;
+                    background-size: 300px 333px, 320px 241px;
+            width: @width-tablet;
+            height: 290px;
+        }
+
+        @media @mq-desktop {
+            background-position: bottom -100px left 60px, bottom -120px right 60px;
+            -webkit-background-size: 381px 423px, 428px 322px;
+                    background-size: 381px 423px, 428px 322px;
+            height: 370px;
```

Comment:
How about adding a min-height media query to adjust the height in wide-but-short windows? This will be a big header on a MacBook Air or landscape tablet.